src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/markup/Table.java

```diff
@@ -275,7 +228,6 @@ public Table setColumnStyles(List<HtmlStyle> styles) {
      * @return  this object
      */
     public Table setRowIdPrefix(String prefix) {
-        rowIdPrefix = prefix;
         return this;
     }
 
```

Comment:
(This comment applies to this entire method. The UI will not let me extend the selection any further)

This method is now effectively empty. Should it be deleted?